### ISSUE TITLE
Write permission error on gem directory

### ISSUE THREAD

**Issue by DeezyE** [beefproject/beef#1938]
#### Environment

What version/revision of BeEF are you using?
0.5.0.0
On what version of Ruby?
2.7
On what browser?
n/a
On what operating system?
Kali 2020.1

#### Configuration

Are you using a non-default configuration?
no
Have you enabled or disabled any BeEF extensions?
no

#### Summary
During install I get a permissions error on the gem folder
`You don't have write permissions for the /var/lib/gems/2.7.0 directory.`
I suspect it's due to the Kali 2020 switch from defaulting to root user
#### Expected Behaviour
install script runs correctly

#### Actual Behaviour
```bash
...
[INFO]  Detecting Ruby environment...
[INFO]  Ruby version 2.7.1 is installed
[INFO]  Updating rubygems...
Updating rubygems-update
Fetching rubygems-update-3.1.4.gem
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /var/lib/gems/2.7.0 directory.
```

#### Steps to Reproduce
New install of Kali 2020.1
run install script

**Comment by bcoles** [comment]
You need to root privileges to install gems (unless you're using a Ruby version management tool such as rvm or rbenv). You can gain root privileges with `su` or `sudo`.

The installer should probably be updated to use `sudo`.

**Comment by DeezyE** [comment]
sudo in the install script is a good idea. Will do!

**Comment by sanjeevsihag** [comment]
> sudo in the install script is a good idea. Will do!

and what about this.  
[INFO]  Detecting Ruby environment...
[INFO]  Ruby version 2.7.0 is installed
[INFO]  Updating rubygems...
ERROR:  While executing gem ... (Gem::RemoteFetcher::UnknownHostError)
    timed out (https://rubygems.org/specs.4.8.gz)

**Comment by jackdwalker** [comment]
@jcrew99 I believe this was solved in #1940. Closing.
